metrics/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+
+services:
+  frontend:
+    image: "mcottontensor/sps-frontend-metrics"
```

Comment:
I assume this is the built image from `metrics/Dockerfile`, if yes, can you add a comment saying just that.